src/components/ProjectSearch.js

```diff
@@ -84,13 +82,18 @@ function sanitizeQuery(query: string): string {
   return query.replace(/\\$/, "");
 }
 
+type FileItem = {
+  setExpanded: (Result, boolean) => any,
+  file: Result,
+  expanded: boolean
+};
+type MatchItem = {
+  expanded: null,
+  match: Match
+};
+
 export class ProjectSearch extends Component<Props, State> {
-  focusedItem: ?{
-    setExpanded?: any,
-    file?: any,
-    expanded?: any,
-    match?: Match
-  };
+  focusedItem: ?FileItem | MatchItem;
```

Comment:
I _think_ this ends up being

```
(?FileItem) | MatchItem
```
but is it intended to be?
```
?(FileItem | MatchItem)
```
I see we do a `if (!this.focusedItem` check elsewhere in the patch.

Reply:
Yeah, that's what I wanted to do, but this did the trick :P